LayoutTests/imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-allowed-for-some-override.https.sub-expected.txt

```diff
@@ -1,3 +1,11 @@
+Blocked access to external URL https://www.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www1.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www1.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www1.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
+Blocked access to external URL https://www1.localhost:9443/feature-policy/resources/feature-policy-allowedfeatures.html
```

Comment:
Any idea why these are new? Is this test usually skipped?

Reply:
It's set to:
imported/w3c/web-platform-tests/feature-policy/feature-policy-frame-policy-allowed-for-some-override.https.sub.html [ DumpJSConsoleLogInStdErr Failure Pass ]

I might revert it, as the above is more or less "skip it"